pkg/types/clusterconfig/cluster_config_aws.go

```diff
@@ -159,6 +158,9 @@ func ValidateRegion(region string) error {
 }
 
 func RegionValidator(region string) (string, error) {
+	if region == "" {
```

Comment:
If required is set to true, then the empty check is no longer necessary. That being said, instead of using `required: true`, I decided to use `minLength: 1`, which has the same effect, except that it says "region cannot be empty". Let me know what you think.

Reply:
I think `required: true` is easier to understand over `minLength: 1` even if it has the same error message.